### ISSUE TITLE
xesmf 0.8.7 not compatible with numpy 2.1.0

### ISSUE THREAD

**Issue by blaylockbk** [pangeo-data/xESMF#388]
In a conda environment, I updated to numpy 2.1.0, but it downgraded xemsf to 0.3.0. Wondered if anyone else gets this behavior and if there is a known reason.

**Comment by aulemahal** [comment]
That's strange! 0.3 was the last version with these 4 characteristics:
- Last to support esmpy/esmf < 8
- Last to support python 3.5
- Last to be released by the original author (also the last version on the original repo)
- For a long time, last to be available on pypi (not the case anymore)

My guess is that for some reason, your esmpy/esmf was downgraded.

**Comment by aulemahal** [comment]
I went back tot his after a similar issue was opened today.

xESMF depends on numba and there are simply no versions of numba that support numpy >= 2.1 (yet). So conda reverted to an earlier version of xESMF.  When I ran mamba requesting for  `xESMF` (nopin) and `numpy>=2.1` in an empty env, I got xESMF 0.6.0 which is indeed the last version without a dependency on numba.

I'll close this issue as unrelated to xESMF. We'll have to wait for an update of numba before being able to use the latest xESMF with numpy >= 2.1, but there are not reasons (yet) to believe xESMF is actually broken.
